apps/dashboard/src/lib/components/Org/Settings/OrgSettings.svelte

```diff
@@ -95,6 +113,14 @@
   function gotoSetting(pathname) {
     goto(`${$currentOrgPath}/settings${pathname}`);
   }
+
+  function resetErrors(_orgName: string) {
+    if (errors.orgName) {
+      errors.orgName = '';
+    }
+  }
+
+  $: resetErrors($currentOrg.name);
```

Comment:
@codiini also I don't think we need this because our general way of reseting error around the app now is letting the user hit the submit button again and then the validation will return an empty object which resets the errors if there isn't any.

Reply:
@rotimi-best Okayy, update pushed.